resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -819,6 +841,7 @@ class MesosCoarseGrainedSchedulerBackendSuite extends SparkFunSuite
     taskScheduler = mock[TaskSchedulerImpl]
     when(taskScheduler.nodeBlacklist).thenReturn(Set[String]())
     when(taskScheduler.sc).thenReturn(sc)
+    when(taskScheduler.nodeBlacklist()).thenReturn(Set[String]())
```

Comment:
This seems to be a duplicate of [line 842](https://github.com/apache/spark/pull/20640/files#diff-74f66a710c02435dedd9ccc581426d69R842).

Reply:
removed, thanks!